cpp/util/etcd_delete.cc

```diff
@@ -64,9 +64,11 @@ void DeleteState::RequestDone(Task* child_task) {
   unique_lock<mutex> lock(mutex_);
   --outstanding_;
 
-  // If a child task has an error, return that error, and do not start
-  // any more requests.
-  if (!child_task->status().ok()) {
+  // If a child task has an error (except for not found, this is close
+  // enough to success), return that error, and do not start any more
+  // requests.
+  if (!child_task->status().ok() &&
```

Comment:
Might be nice to still treat `NOT_FOUND` as a legitimate error when it's from a regular (i.e. with handle) delete; failing to delete something you really expected to be there could be "interesting" info.